modules/rhcos-enabling-multipath.adoc

```diff
@@ -1,122 +1,65 @@
-----
-$ oc get MachineConfig
-----
+* For a single multipath device, use the path `/dev/mapper/mpatha`, for example:
 +
-.Example output
 [source,terminal]
 ----
-NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
-00-master                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-00-worker                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-kargs-mpath                              52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             105s
-99-master-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-ssh                                                                                 3.2.0             40m
-99-worker-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-worker-ssh                                                                                 3.2.0             40m
-rendered-master-23e785de7587df95a4b517e0647e5ab7   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-rendered-worker-5d596d9293ca3ea80c896a1191735bb1   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
+$ coreos-installer install /dev/mapper/mpatha \ <1>
+--append-karg rd.multipath=default \
+--append-karg root=/dev/mapper/dm-mpath-root
 ----
-
-. Check the nodes:
+<1> Indicates the path of the single multipathed device.
 +
-[source,terminal]
-----
-$ oc get nodes
-----
+* For multiple multipath devices, or to be more explicit, it is recommended that the names of the multipath devices are aliased to a World Wide Name (WWN) symlink when you append the kernel arguments , such as `/dev/disk/by-label`. For example:
 +
-.Example output
 [source,terminal]
 ----
-NAME                           STATUS                     ROLES    AGE   VERSION
-ip-10-0-136-161.ec2.internal   Ready                      worker   28m   v1.20.0
-ip-10-0-136-243.ec2.internal   Ready                      master   34m   v1.20.0
-ip-10-0-141-105.ec2.internal   Ready,SchedulingDisabled   worker   28m   v1.20.0
-ip-10-0-142-249.ec2.internal   Ready                      master   34m   v1.20.0
-ip-10-0-153-11.ec2.internal    Ready                      worker   28m   v1.20.0
-ip-10-0-153-150.ec2.internal   Ready                      master   34m   v1.20.0
+$ coreos-installer install /dev/disk/by-id/<wwn_ID> \ <1>
```

Comment:
Should this be

```suggestion
$ coreos-installer install /dev/disk/by-id/wwn-<wwn_ID> \ <1>
```

instead, and then the `<1>` below is just `0xx194e957fcedb4841` for the WWN ID?

Reply:
Resolved